### PR TITLE
Improve Hot Plug Disk feature page

### PR DESCRIPTION
**Pull request by mureinik**
Unify and improve the Hot Plug Disks pages. This PR handles point 290 of issue #37 .

### CODE REVIEW

**Comment by mureinik** [comment]
@aaviram please review? Specifically, the dictionary passed to VDSM seems a bit odd to me, not sure if it's outdated or not.

File: source/develop/release-management/features/storage/hotplugdisk.html.md [hunk 1]
```diff
@@ -96,26 +73,5 @@ A call to vdsm will be done in synchronious way
...
-Is there upstream documentation on this feature, or notes you have written yourself? Link to that material here so other interested developers can get involved. Links to RFEs.
-
-### Comments and Discussion
-
-Add a link to the "discussion" tab associated with your page. This provides the ability to have ongoing comments or conversation without bogging down the main feature page
-
-### Open Issues
-
-NA
+New vdsm errors will be added: `FailedToPlugDisk` (45) and `FailedToUnPlugDisk` (46).
```
Comment (aaviram) [inline source/develop/release-management/features/storage/hotplugdisk.html.md]: The errors are "HotplugDiskFailed" and "HotunplugDiskFailed"

File: source/develop/release-management/features/storage/hotplugdisk.html.md [hunk 2]
```diff
@@ -17,65 +17,42 @@ feature_status: Released
 
 ### Summary
 
-Allow to hot add and hot remove of a virtio-blk disks from or to running guest.
-
-### Owner
-
-*   Michael Kublin
-*   Email: mkublin@redhat.com
```
Comment (aaviram) [inline source/develop/release-management/features/storage/hotplugdisk.html.md]: This is old code, but sure about removing the owner?

File: source/develop/release-management/features/storage/hotplugdisk.html.md [hunk 3]
```diff
@@ -17,65 +17,42 @@ feature_status: Released
...
+1. The guest OS supports the operation
+2. VirtIO/VirtIO-SCSI disk
+3. Disk should be plugged
+4. VM should be up
 
-In order to perform an operation new verbs will be added at VDSM side:
-hotplug and hotunplug with the following dictionary to pass:
-A call to vdsm will be done in synchronious way
+In order to perform this operation new verbs will be added to the VDSM side:
+`hotplug` and `hotunplug` with the following argument dictionary:
```
Comment (aaviram) [inline source/develop/release-management/features/storage/hotplugdisk.html.md]: VM.hotplugDisk , VM.hotunplugDisk.
The parameters are not exactly a dictionary, it is VDSM types as described in the schema-
hotplugDisk gets:
{'vmID': 'UUID', 'params': 'HotplugDiskParams'}

hotunplugDisk gets:
{'vmID': 'UUID', 'params': 'HotplugDiskParams'}

These types have other types inside, and it is all detailed in VDSM's schema. anyway it is not what's written here. Question is if we want to write it all down here or refer to elsewhere.

File: source/develop/release-management/features/storage/hotplugdisk.html.md (hunk 2 above)
Comment (mureinik) [inline source/develop/release-management/features/storage/hotplugdisk.html.md]: This email no longer exists, so I'm not sure what's the point of keeping it, but you're right - such decisions should be made in discussions, not patches. I'll revert this change.

File: source/develop/release-management/features/storage/hotplugdisk.html.md (hunk 3 above)
Comment (mureinik) [inline source/develop/release-management/features/storage/hotplugdisk.html.md]: I agree there's very little point in having this here.

File: source/develop/release-management/features/storage/hotplugdisk.html.md (hunk 1 above)
Comment (mureinik) [inline source/develop/release-management/features/storage/hotplugdisk.html.md]: Will fix, thanks.

**Comment by mureinik** [comment]
@aaviram please review the updated patch series, thanks!

**Comment by aaviram** [comment]
+1

File: source/develop/release-management/features/storage/hotplugdisk.html.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+title: Hot Plug and Unplug Disk
+category: template
+authors: derez, mkublin, ykaul
+wiki_category: Template
+wiki_title: Features/DetailedHotPlug
+wiki_revision_count: 25
+wiki_last_updated: 2014-07-13
```
Comment (sandrobonazzola) [inline source/develop/release-management/features/storage/hotplugdisk.html.md]: I may be wrong, but above wiki_* metadata are meaningless on new files, there's no corresponging wiki page on old wiki instance. @garrett ?

Comment (mureinik) [inline source/develop/release-management/features/storage/hotplugdisk.html.md]: @garrett can you chime in here please?

**Comment by garrett** [comment]
Yes, wiki_* is almost meaningless for new files. The exception is `wiki_title`, which may be used as an alternate way to link a file (else all the old wiki links break or they would all have to be migrated).

You can have a page without any wiki_* metadata, including `wiki_title`, of course.

Tutle, category, and authors are all still required — especially for feature pages.

**Pull request merged**